CHANGELOG.md

```diff
@@ -1,10 +1,172 @@
 TODO: This document was manually maintained so might be incomplete. The
 automation effort is tracked in
-https://github.com/kubernetes/client-go/issues/234.
+https://github.com/kubernetes/test-infra/issues/5843.
 
 Changes in `k8s.io/api` and `k8s.io/apimachinery` are mentioned here
 because `k8s.io/client-go` depends on them.
 
+# v8.0.0
+
+**Breaking Changes:**
+
+* `KUBE_API_VERSIONS` has been removed. Pleas use the `runtime-config` flag instead.
```

Comment:
How does this apply to the client? runtime-config does not exist as a flag. But I guess we had support for the env var before? Probably.

Reply:
fixed